### ISSUE TITLE
Tutorial example for PersonView and graphql not working

### ISSUE THREAD

**Issue by imichael** [edgedb/edgedb#620]
For this graphql example in the tutorial

```
{
    PersonView(order: {last_name: {dir: ASC}}) {
        name
        acted_in { title }
    }
}
```

I am getting this error in the browser

```
{
  "errors": [
    {
      "message": "InvalidReferenceError: reference to a non-existent schema item: Person",
      "locations": [
        {
          "line": -1,
          "column": -1
        }
      ]
    }
  ]
}
```
In the edgedb client, it seems to be working:

```
tutorial> select PersonView { name, acted_in: { title } };
{
    Object { name: 'Denis Villeneuve', acted_in: {} },
    Object { name: 'Harrison Ford', acted_in: {Object { title: 'Blade Runner 2049' }} },
    Object { name: 'Ryan Gosling', acted_in: {Object { title: 'Blade Runner 2049' }} },
    Object { name: 'Ana de Armas', acted_in: {Object { title: 'Blade Runner 2049' }} },
    Object { name: 'Jason Momoa', acted_in: {Object { title: 'Dune' }} },
    Object { name: 'Oscar Isaac', acted_in: {Object { title: 'Dune' }} },
    Object { name: 'Zendaya', acted_in: {Object { title: 'Dune' }} }
}
tutorial> select (PersonView.name, PersonView.acted_in.title);
{
    ('Harrison Ford', 'Blade Runner 2049'),
    ('Ryan Gosling', 'Blade Runner 2049'),
    ('Ana de Armas', 'Blade Runner 2049'),
    ('Jason Momoa', 'Dune'),
    ('Oscar Isaac', 'Dune'),
    ('Zendaya', 'Dune')
}
tutorial>

```

**Comment by vpetrovykh** [comment]
Thanks for reporting this. It seems like an issue with resolving the default module. For the moment you can use the following view instead (drop old view and make a new one):
```
DROP VIEW PersonView;
CREATE VIEW PersonView := default::Person {
    acted_in := default::Person.<cast[IS default::Movie]
};
```
The proper fix is coming soon.

**Comment by elprans** [comment]
I just pushed the new revision of the alpha 1 packages with the fix (e57b5b62f)  for this backported.
